docs/source/developer.rst

```diff
@@ -131,3 +131,75 @@ Documenting Changes
 
 When you add a public method, please document it in the usage docs and the test
 mode docs. Follow the format for already-existing methods.
+
+Uploading to PyPi
+-----------------
+
+When new features are added or bugs are fixed it is necessary to push Pymesync
+to `PyPi`_ so users can ``pip install`` those changes. This can only be done by
+owners and maintainers of Pymesync, listed below. Email support@osuosl.org if
+you believe a new version is warranted.
+
+====================== ==========
+Developer              Role
+====================== ==========
+Matthew Johnson (mrsj) Owner
+Ken Lett (kennric)     Owner
+Alex Taylor (subnomo)  Maintainer
+====================== ==========
+
+There are several steps that a developer must take before submitting Pymesync
+to PyPi:
+
+#. Follow the article `How to submit a package to PyPi`_ by Peter Downs to
+   create accounts and set up your local configuration file.
+
+#. Open a PR and merge the ``develop`` branch into ``master``. This should be
+   reviewed by an owner or maintainer to ensure the update is necessary.
+
+#. Directly on the ``master`` branch, an owner or maintainer should bump the
+   ``version`` in ``setup.py`` following the
+   `Semantic Versioning Specification`_ (SemVer). `Tag that commit`_ with the
+   version number.
+
+#. Upload Pymesync to `PyPi Test`_ first to make sure that everything is
+   working.
+
+   ::
+
+     (venv) $ python setup.py register -r pypitest
+     [... register to pypitest success ...]
+     (venv) $ python setup.py sdist upload -r pypitest
+     [... upload to pypitest success ...]
+
+   .. note::
+
+     ``pypitest`` is a configuration set in ``.pypirc`` from step 1.
+
+   Now visit https://testpypi.python.org/pypi, search for ``pymesync``, and
+   make sure the version is up to date.
+
+#. Upload Pymesync to PyPi.
+
+   ::
+
+     (venv) $ python setup.py register -r pypi
+     [... register to pypi success ...]
+     (venv) $ python setup.py sdist upload -r pypi
+     [... upload to pypi success ...]
+
+   .. note::
+
+     ``pypi`` is a configuration set in ``.pypirc`` from step 1.
+
+   Now visit https://pypi.python.org/pypi to make sure the version is up to
+   date.
+
+#. Inform your users that Pymesync has been updated.
```

Comment:
Is there a set way to do this?
Rephrasing it to be something like *Inform pymesync users that there has been an update in whatever way is the current standard.*, or something like that.